docs/en/api/FS-API.md

```diff
@@ -1,36 +1,38 @@
 ---
 layout: global
-title: Filesystem API
+title: File System API
```

Comment:
Our [documentation convention](https://docs.alluxio.io/os/user/edge/en/contributor/Documentation-Conventions.html#terminology-table) says that `filesystem` is wrong.